code/modules/mob/emote.dm

```diff
@@ -85,3 +85,17 @@
 						riding_datum.force_dismount(M)
 			else
 				L.unbuckle_all_mobs()
+/datum/emote/thud
+	key = "thud"
+	key_third_person = "thuds"
+	hands_use_check = TRUE
+	mob_type_allowed_typecache = list(/mob/living, /mob/dead/observer)
+	mob_type_ignore_stat_typecache = list(/mob/dead/observer)
+
+/datum/emote/thud/run_emote(mob/user, params,  type_override, intentional)
+	. = ..()
+	if(.)
+		if(isobserver(user) && prob(90)) //they know a little about rolling down in the deep
+			user.playsound_local(get_turf(user), 'sound/voice/vinethud.ogg', 50, TRUE)
+			return
+		playsound(user, 'sound/voice/vinethud.ogg', 50, 1, -1)
```

Comment:
So, the vine thud is funny and all but in seriousness i don't think that *thud will be good for us. If you can remove the emote or make it an admin-only thing we can get this merged.

Reply:
commented out the emote for later, at least until i learn how to make shit admin only lol. having no code is better than making shitcode.